soul-admin/src/main/resources/application.yml

```diff
@@ -29,7 +29,7 @@ spring:
   datasource:
     url: jdbc:mysql://localhost:3306/soul?useUnicode=true&characterEncoding=utf-8
     username: root
-    password:
+    password: 
```

Comment:
revert

Reply:
Invalid submission，more submitted application.yml

Reply:
Personal profiles will not be submitted in the future